docs/installation/advanced/seafile.md

```diff
@@ -1,31 +1,26 @@
----
-status: wip
----
-
 # Integrate with Seafile
 
 <!-- md:version 4.0 -->
 <!-- md:flag enterprise -->
 
-When integrating SeaTable and Seafile, users can add files from Seafile libraries in file columns and users can transfer files from SeaTable to Seafile.
+When integrating SeaTable and Seafile, users can add files from Seafile libraries in file columns.
 
 ### Scope of the Connection
 
 The Seafile connection to SeaTable enables you to
 
 - Choose a file as attachment from the Seafile library in SeaTable
-- Save attachments in a SeaTable base to your Seafile library
```

Comment:
This was mentioned in https://forum.seatable.com/t/seafile-link-to-my-seatable-database/366/2 (as not yet implemented), but I couldn't figure out whether this is still the case. I tried it with a form, but there was no option to configure the storage location.